afs/config_handler.py

```diff
@@ -98,6 +94,12 @@ def get_data(self):
 
         :return: DataFrame type. Data from REQUEST and rename column name.
         """
+        try:
+            data = json.loads(self.REQUEST)['body']['data']
+            self.data = DataFrame.from_dict(data)
+        except Exception as e:
+            raise AssertionError('Request contains no key named "data", or cannot transform to dataframe.')
```

Comment:
Considering to add another error case when `self.REQUEST` is not JSON string? Even this checked by `set_kernel_gateway()` but I can still run `get_param()` brfore `set_kernel_gateway()`, right? And the current error message also covered by `KeyError` and `ValueError`, shell we use `AssertionError` here?

Reply:
There should be a if-else to check the REQUEST data is empty.
And the error type would be change the respond type